Formula/gcc.rb

```diff
@@ -34,12 +34,7 @@ class Gcc < Formula
 
   # The bottles are built on systems with the CLT installed, and do not work
   # out of the box on Xcode-only systems due to an incorrect sysroot.
-  pour_bottle? do
-    on_macos do
-      reason "The bottle needs the Xcode CLT to be installed."
-      satisfy { MacOS::CLT.installed? }
-    end
-  end
+  pour_bottle? reason: :clt_required
```

Comment:
```suggestion
  pour_bottle? requirement: :clt
```
This ship might've sailed already, but I just realised this might've been nicer 😅

Reply:
Hmm maybe. I mean, it's only been in a tagged release for a few hours so changing it without deprecation _probably_ wouldn't cause an issue...

Reply:
Let's change it. This makes more sense to me. The way it looks currently makes it sound like `:clt_required` is the reason to pour the bottle, which doesn't really make sense.

Reply:
And if we have more formula that require a default prefix for the bottle:
```
pour_bottle? requirement: :default_prefix
```

Reply:
Yeah, that makes sense to me. I did think about adding a `default_prefix` but we decided against it because it would only be used in one instance and most default prefix requirements should be caught by `brew bottle` anyway.

Reply:
Go ahead and open a PR if you'd like I won't be able to for a bit

Reply:
https://github.com/Homebrew/brew/pull/11408